src/nvim/fileio.c

```diff
@@ -1883,16 +1883,20 @@ readfile (
       xfree(keep_msg);
       keep_msg = NULL;
       msg_scrolled_ign = TRUE;
-      p = msg_trunc_attr(IObuff, FALSE, 0);
+
+      if (!read_stdin && !read_buffer) {
+        p = msg_trunc_attr(IObuff, FALSE, 0);
```

Comment:
@oni-link Thank you. @DarkDeepBlue see https://github.com/neovim/neovim/commit/c86caf7e41c77f85861fc92e1ef1462508d74b24